cmd/fake-qemu-process/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["fake-qemu.go"],
+    importpath = "kubevirt.io/kubevirt/cmd/fake-qemu-process",
+    visibility = ["//visibility:private"],
+)
+
+go_binary(
+    name = "fake-qemu-process",
+    embed = [":go_default_library"],
```

Comment:
I wonder if the pattern of embedding library that defines binary is useful in this case.